### PR TITLE
Add stupid roomba, preliminary support for empty sim (not finished, so no launchfile support)

### PR DESCRIPTION
**Pull request by amiller27**

### CODE REVIEW

**Comment by asaba96** [comment]
is this ready to be reviewed?

**Comment by amiller27** [comment]
Yes

File: scripts/stupid_roomba.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import rospy
+import tf2_ros as tf2
+
+from geometry_msgs.msg import TransformStamped
+from iarc7_msgs.msg import OdometryArray
+from nav_msgs.msg import Odometry
+
+if __name__ == '__main__':
+    rospy.init_node('stupid_roomba')
```
Comment (asaba96) [inline scripts/stupid_roomba.py:11]: considering our code is public do we really want to start the convention of using childish names for things?

**Comment by asaba96** [comment]
Also, has this been tested thoroughly?

**Comment by amiller27** [comment]
It has been tested adequately

**Review by asaba96 — APPROVED** [review]
Lgtm mostly

**Pull request merged**